ros2topic/ros2topic/verb/pub.py

```diff
@@ -36,6 +36,22 @@
 MsgType = TypeVar('MsgType')
 
 
+def positive_int(inval):
+    ret = int(inval)
+    if ret < 0:
+        # The error message here gets completely swallowed by argparse
+        raise ValueError('Value must be positive')
+    return ret
```

Comment:
Both functions use the term `positive` but handle differently. That seems pretty confusing to me. Positive integers do not include zero. So I would suggest to rename this function to `non_negative_int` and update the error message accordingly.